buildSrc/src/main/kotlin/net/skyscanner/backpack/tokens/BpkColors.kt

```diff
@@ -178,29 +173,6 @@ private fun deprecationProperty(): PropertySpec =
 @OptIn(ExperimentalStdlibApi::class)
 private fun String.toHexColor() = uppercase().run { "0x${substring(7)}${substring(1, 7)}" }
 
-private fun toStaticCompose(
```

Comment:
So, we're not generating static compose colors at all, right?

Reply:
nope - we're meant to only use semantic ones anyways. they're also not used in the app at all. with the view system we've got historic colour usage, but in compose we've started from scratch